dayPlanner.css

```diff
@@ -37,4 +37,40 @@ tbody th {
 
 tbody tr {
     cursor: pointer;
-}
\ No newline at end of file
+}
+
+#settingBox {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(0,0,0);
+    background-color: rgba(0, 0, 0, 0.4);
+}
+
+.settingBoxContent {
+    background-color: #fefefe;
+    margin: 15% auto;
+    padding: 20px;

```

Comment:
- burada da 4 kenar içinde 20 pixellik bi boşluk koyuyoruz ama bu boşluk kenarlarla ögenin içindeki içeriğin arasındaki boşluğu ifade ediyor.